superset-frontend/src/components/TableSelector.jsx

```diff
@@ -103,11 +103,11 @@ export default class TableSelector extends React.PureComponent {
     });
   }
 
-  onDatabaseChange(db, selectChangeMeta) {
-    return this.changeDataBase(db);
+  onDatabaseChange(db, force) {
```

Comment:
@ktmud this is the only instance where instead of deleting the unused variable, I tried to fix logic that seemed to be missing. Could you confirm if this is a correct change?

Reply:
look good to me.